$MirOS: ports/net/djbdns/patches/patch-src_query_h,v 1.2 2005/12/14 23:28:37 tg Exp $
--- src/query.h.orig	Sun Feb 11 21:11:23 2001
+++ src/query.h	Sat May 30 21:47:38 2009
@@ -14,16 +14,29 @@ struct query {
   char *name[QUERY_MAXLEVEL];
   char *control[QUERY_MAXLEVEL]; /* pointing inside name */
   char *ns[QUERY_MAXLEVEL][QUERY_MAXNS];
+#ifndef DJB_V6ONLY
   char servers[QUERY_MAXLEVEL][64];
+#else
+  char servers[QUERY_MAXLEVEL][512];
+#endif /* DJB_V6ONLY */
   char *alias[QUERY_MAXALIAS];
   uint32 aliasttl[QUERY_MAXALIAS];
+#ifndef DJB_V6ONLY
   char localip[4];
+#else
+  char localip[16];
+  uint32 scope_id;
+#endif /* DJB_V6ONLY */
   char type[2];
   char class[2];
   struct dns_transmit dt;
 } ;
 
+#ifndef DJB_V6ONLY
 extern int query_start(struct query *,char *,char *,char *,char *);
+#else
+extern int query_start(struct query *,char *,char *,char *,char *,unsigned int);
+#endif /* DJB_V6ONLY */
 extern void query_io(struct query *,iopause_fd *,struct taia *);
 extern int query_get(struct query *,iopause_fd *,struct taia *);
 
